docs/react-hook-form.md

```diff
@@ -161,55 +161,52 @@ const MyComponent = () => {
 }
 ```
 
```

Comment:
Following my previous comment, all this part is not inline with what we're doing in hublo-front.
Now `zod` deals with already translated messages.
Maybe we should talk about it, especially with Thomas if you don't agree.